GitUI/UserControls/NativeTreeView.cs

```diff
@@ -7,5 +7,11 @@ protected override void CreateHandle()
             base.CreateHandle();
             NativeMethods.SetWindowTheme(Handle, "explorer", null);
         }
+
+        public new bool DoubleBuffered
```

Comment:
Is there a reason we don't just set this property at the time we construct an instance? This way we won't need to shadow the property and set it explicitly....

Reply:
No reason I suppose.. this option is "an option" in the first place because it has performance impact (probably minuscule in our case). I thought it would be better to have it set explicitly but this is not a strong opinion, I am fine with setting it on the base control. Should I change it?

Reply:
If this no compelling reason then I'd avoid unnecessary API and do it at the .ctor.